includes/admin/stripe-settings.php

```diff
@@ -148,6 +149,14 @@
 			'default'     => '44',
 			'desc_tip'    => true,
 		),
+		'payment_request_button_label' => array(
+			'title'       => __( 'Payment Request Button Label', 'woocommerce-gateway-stripe' ),
+			'label'       => __( 'Button Label', 'woocommerce-gateway-stripe' ),
+			'type'        => 'text',
+			'description' => __( 'Enter the custom text you would like the button to have.', 'woocommerce-gateway-stripe' ),
+			'default'     => 'Buy now',
```

Comment:
Please make the default value translatable as well.

Reply:
Added translation in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1197/commits/4594c68503cb2bba40f9204b970121ddb280cf5a